modin/backends/pandas/parsers.py

```diff
@@ -11,30 +11,87 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""This module houses parser classes (classes that are used for data parsing on the workers)
+and util functions for handling parsing results.
+
+"""
+
 from collections import OrderedDict
 from io import BytesIO
 import numpy as np
 import pandas
 from pandas.core.dtypes.cast import find_common_type
 from pandas.core.dtypes.concat import union_categoricals
 from pandas.io.common import infer_compression
+from pandas.util._decorators import Appender
```

Comment:
I suggest doing it like

```python
from pandas.util._decorators import Appender as DocstringAppender
```

to make it clear what `@Appender` construct does down below in the code

Reply:
According to your recommendation `@Appender` decorator was replaced with more suitable `@doc` decorator.